tasks/UsblDock.cpp

```diff
@@ -29,14 +29,11 @@ bool UsblDock::configureHook()
     if (! UsblDockBase::configureHook())
         return false;
 
-    if (!_io_port.get().empty())
+    if(_interface.get() != ETHERNET)
```

Comment:
I change my mind. Better having it simple without this property. The case of two connectors per device is not considered in the near future, I think.